docs/docs/testing/helpers.md

```diff
@@ -152,6 +152,12 @@ await elementUpdated(el);
 expect(el.foo).to.equal('bar');
 ```
 
+All of these helpers must be imported from `@open-wc/testing/index-no-side-effects.js`, e.g.
+
+```js
+import { nextFrame, aTimeout, waitUntil } from '@open-wc/testing/index-no-side-effects.js';
```

Comment:
```suggestion
import { nextFrame, aTimeout, waitUntil } from '@open-wc/testing';
```

Reply:
Should both package names be the same here? e.g. should "All of these helpers must be imported from `@open-wc/testing/index-no-side-effects.js`" be "All of these helpers must be imported from `@open-wc/testing`"?

Reply:
yes

Reply:
Oops, I resolved this when I meant to commit it. Now I can't commit the suggestion 😩

Reply:
Oh, looks like it was committed, perfect.